### ISSUE TITLE
sigma_cont() bug when >3 percents given

### ISSUE THREAD

**Issue by AstroRobin** [MBravoS/splotch#21]
There is a bug in `sigma_cont()` wherever the user gives more than 3 percents. The colour array being produced contains RGBA arrays instead of single colour strings. The result of taking the len(c) is a value with 2 dimensions which is breaking one the if statements somewhere later.

Minimal code that fails.

```
data = np.random.normal(5.0,1.5,size=[2,1000])

fig, axes = plt.subplots()
splt.sigma_cont(data[0],data[1],percent=[40,50,75,90])
plt.show()
```

**Comment by MBravoS** [comment]
Solved since version 0.3.0.4.

**Comment by AstroRobin** [comment]
Go to bed!
